Something/src/test/java/org/example/NameListsTest.java

```diff
@@ -0,0 +1,39 @@
+package org.example;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+public class NameListsTest {
+    NameLists nameLists;
+    @BeforeEach
+    public void setUp(){
+        nameLists = new NameLists();
+        nameLists.setNamesA("John");
+        nameLists.setNamesA("Brian");
+        nameLists.setNamesA("Steve");
+
+        nameLists.setNamesB("Brian");
+        nameLists.setNamesB("Barry");
+        nameLists.setNamesB("Tyrone");
+        nameLists.setNamesB("Hector");
+        nameLists.setNamesB("Jimmy");
+    }
+    @Test
+    public void testReturnNamesA(){
```

Comment:
We tend not so test methods that just get or just set attributes of a class, unless those methods do more than just setting or getting.